### ISSUE TITLE
A question

### ISSUE THREAD

**Issue by arianaa30** [yyakult/DASH-VR-Player#2]
It seems like a great work:) 
So what features have you added to EleVR?

**Comment by yyakult** [comment]
hi, actually I just follow the instructions to generate MPD file (then the video source written in html5 is like src="Manifest.mpd") and add dash.js player in <video></video>. I will add more details these days. : )

**Comment by arianaa30** [comment]
I see. So basically you have added the DASH feature into the VR player, right?

**Comment by yyakult** [comment]
Yep.eleVR is a great VR player but it cannot play the video format in DASH and it only support two

**Comment by yyakult** [comment]
oh, sorry I clicked close and comment...actually eleVR just support two projections (equirectangular and equirectangular 3D). My future work is let the player support cubic projection if you are interested in it we can talk more often : )

**Comment by arianaa30** [comment]
Oh, yes. Cubic format support would be great! Don't you have an Android version for that? I wish work work was an Android app!

**Comment by yyakult** [comment]
:( currently nope. I think a web player is more flexible towards different operating system. eleVR works well with Android system now.
